src/translators/crate.py

```diff
@@ -234,14 +234,18 @@ def _insert_entities_of_type(self,
                 original_attrs[col] = (attr, attr_t)
 
                 if attr_t not in NGSI_TO_CRATE:
-                    supported_types = ', '.join(NGSI_TO_CRATE.keys())
-                    msg = ("'{}' is not a supported NGSI type. "
-                           "Please use any of the following: {}. "
-                           "Falling back to {}.")
-                    self.logger.warning(msg.format(
-                        attr_t, supported_types, NGSI_TEXT))
+                    # if attribute is complex assume it as an NGSI StructuredValue
+                    if self._attr_is_structured(e[attr]):
```

Comment:
it is a missing feature... at the time being, we support only array of strings (which I agree is limited)

Reply:
Okie, but should we implement it as part of this PR? The reason why I'm saying this is that if we want to support dictionaries then we should do it consistently across the board, to avoid clients getting confused about when they can use a dictionary...

Reply:
opened a separate issue (#147) for this to avoid putting too many things in this PR :-)